baseframe/forms/form.py

```diff
@@ -38,6 +38,7 @@
     'AnnotatedTextField': bfields.AnnotatedTextField,
     'MarkdownField': bfields.MarkdownField,
     'ImageField': bfields.ImgeeField,
+    'EnumField': bfields.EnumField,
```

Comment:
No EnumField here. This is for the form generator, which does not have access to enums.